helm/dex-operator/Chart.yaml

```diff
@@ -7,4 +7,4 @@ icon: https://s.giantswarm.io/app-icons/dex/2/icon_light.svg
 apiVersion: v1
 annotations:
   application.giantswarm.io/team: "bigmac"
-  config.giantswarm.io/version: 1.x.x
+  config.giantswarm.io/version: auth-cm
```

Comment:
Is that intentional?

Reply:
I used it for testing. :D I'll get rid of it before merge